source/features/mark-private-orgs.tsx

```diff
@@ -11,8 +11,8 @@ async function init(): Promise<false | void> {
 		return false;
 	}
 
-	let publicOrgs = await api.v3(`users/${getUsername()}/orgs`);
-	publicOrgs = publicOrgs.map((orgData: AnyObject) => `/${orgData.login}`);
+	const response = await api.v3(`users/${getUsername()}/orgs`);
+	const publicOrgs = response.map((orgData: AnyObject) => `/${orgData.login}`) as string[];
```

Comment:
Do you really need the type assertion here? I would assume TS would be able to infer the string.

Even this would be better:

```suggestion
	const publicOrgs: string[] = response.map((orgData: AnyObject) => `/${orgData.login}`);
```

Reply:
> I would assume TS would be able to infer the string.

Sadly not. I think `publicOrgs` was an `any` here. I typed it because `no-unnecessary-condition` gave me an error on the `.includes()` condition